tokens/figma/update-skins.mjs

```diff
@@ -0,0 +1,165 @@
+import {
+  updateCollections,
+  updateOrCreateVariables,
+  updateOrCreateVariableModeValues,
+} from "./utils/figma-utils.mjs";
+
+import {
+  COLLECTION_NAMES,
+  MODE_NAMES,
+} from "./utils/constants.mjs";
+
+import {
+  getFigmaData,
+  postFigmaVariables,
+} from "./utils/api-request.mjs";
+
+import { getPaletteVariables } from "./variables.mjs";
+
+const collectionNames = [
+  COLLECTION_NAMES.PALETTE,
+];
+
+async function updatePalette(
+  jsonData,
+  brand,
+  FILE_KEY
```

Comment:
As mentioned before, I'd remove all these parameters in the functions. They shouldn't be passed as parameters, given that they are constants.

In this case, I'd just keep the object that maps a brand to it's file key somewhere so that you can access it from here (without having to pass it as parameter in several function calls). This will make your life easier for future refactors

Reply:
I believe i resolved most of the cases the constants shouldn't be as a parameter of a function.